pages/mesosphere/dcos/1.13/release-notes/1.13.8/index.md

```diff
@@ -0,0 +1,57 @@
+---
+layout: layout.pug
+navigationTitle: Release notes for 1.13.8
+title: Release notes for 1.13.8
+menuWeight: 2
+excerpt: Release notes for DC/OS 1.13.8, including Open Source attribution, and version policy.
+---
+DC/OS&trade; 1.13.8 was released on 19 March 2020.
+
+<p class="message--warning"><strong>WARNING:</strong> The DC/OS 1.13.8 release has a critical error in its networking stack and we recommend that you do NOT download or install it. Instead, download and install the 1.13.9 patch [here](/mesosphere/dcos/1.13/release-notes/1.13.9/).</p> 
```

Comment:
Can we call this a release and not a patch ?

Reply:
done!